### PR TITLE
Modernize instructions in how-to-docs.html

### PR DESCRIPTION
**Pull request by jwakely**

### CODE REVIEW

File: how-to-docs.html [hunk 1]
```diff
@@ -8,75 +8,87 @@
...
 
 <h1>How To Instructions</h1>
-<p>These instruction assume you going to keep the issues lists source files in a directory named &quot;issues&quot;, 
-and the issues lists generated HTML files in a directory named &quot;issues-gh-pages&quot;.</p>
-<blockquote>
-<p>Keeping the source files and generated HTML files in separate directories 
-falls out from the GitHub automatic web page approach: files in a branch named &quot;gh-pages&quot; 
-are automatically published by GitHub as web pages.</p>
-</blockquote>
+<p>These instruction assume you going to keep the issues lists source files in a directory named &quot;issues&quot;.
```
Comment (CaseyCarter) [inline how-to-docs.html:15]: For consistency:
```suggestion
<p>These instructions assume you keep the issues lists source files in a directory named <code>issues</code>.
```

File: how-to-docs.html [hunk 2]
```diff
@@ -8,75 +8,87 @@
...
+The issues are stored in XML files, one file per issue.
```
Comment (CaseyCarter) [inline how-to-docs.html:16]:
```suggestion
Each issue is stored in an XML file.
```

File: how-to-docs.html [hunk 3]
```diff
@@ -8,75 +8,87 @@
...
+The software in the <code>src</code> directory builds programs
+that read the XML files and generates the HTML issues lists.
```
Comment (CaseyCarter) [inline how-to-docs.html:18]:
```suggestion
The software in the <code>src</code> directory consists of programs
that read the XML files and generate the HTML issues lists.
```

File: how-to-docs.html [hunk 4]
```diff
@@ -8,75 +8,87 @@
...
+generates new HTML files and commits them to the <code>gh-pages</code> branch.
+The content of the <code>gh-pages</code> branch is automatically deployed
+to <a href="https://cplusplus.github.io/LWG/">https://cplusplus.github.io/LWG/</a>.
+</p>
 
 <h2>Prerequisites</h2>
 <ul>
   <li>Git</li>
   <li>A C++17 compiler. Clang 9 works. GCC 8.x works for unix-like systems, but GCC 9.3 is recommended (because its <code>std::filesystem</code> supports Windows, and no additional <code>-lstdc++fs</code> linker option is needed).</li>
   <li>Windows is a prerequisite to use the Windows <code>.bat</code> scripts, or
```
Comment (CaseyCarter) [inline how-to-docs.html:30]: These substatements are in fact both true.
```suggestion
  <li>Windows is a prerequisite to use the Windows <code>.bat</code> scripts;
```

File: how-to-docs.html [hunk 5]
```diff
@@ -8,75 +8,87 @@
...
+whenever the source code (in <code>issues/src</code>) changes.
+(This isn't needed when using <code>make</code> because the makefile
+will automatically do <code>make pgms</code> as a prerequisite of
+<code>make lists</code>.)
+</p>
 
 <h2>Add or change issues</h2>
 <p>There is one source file per issue, in <code>issues/xml</code>. The file
-<code>issues/xml/lwg-template.xml</code> can be used as a template to start a 
+<code>issues/xml/lwg-template.xml</code> can be used as a template to start a
```
Comment (CaseyCarter) [inline how-to-docs.html:73]:
```suggestion
<code>issues/xml/lwg-template.xml</code> is a template for a
```

File: how-to-docs.html [hunk 6]
```diff
@@ -8,75 +8,87 @@
...
 new issue. Use any text editor to edit the files.</p>
```
Comment (CaseyCarter) [inline how-to-docs.html:74]: I sincerely hope anyone who makes it here knows how to edit text files.
```suggestion
new issue.</p>
```

File: how-to-docs.html [hunk 7]
```diff
@@ -8,75 +8,87 @@
...
-edit-as-needed
-git commit -a -m&quot;your commit message here&quot;</pre>
-</blockquote>
-<p>Verify your changes had the desired effect by regenerating the 
-issues lists and inspecting your changes:</p>
-<blockquote>
-  <pre>cd issues
-bin\build_lists</pre>
+  <pre>cd xml
+<em># edit the XML files or add new XML files</em>
```
Comment (CaseyCarter) [inline how-to-docs.html:78]: This current directory change is silly and invalidates the commands below.
```suggestion
  <pre><em># edit the XML files or add new XML files</em>
```

File: how-to-docs.html [hunk 8]
```diff
@@ -8,75 +8,87 @@
...
+<em># ...</em>
+<em># stage the edited XML files (and any new ones) to be committed:</em>
+git add xml/issues*.xml
+git commit -m &quot;your commit message here&quot;</pre>
```
Comment (CaseyCarter) [inline how-to-docs.html:82]: What's with the `&quot;` entities in this file?
```suggestion
git commit -m "your commit message here"</pre>
```

**Review by CaseyCarter — CHANGES_REQUESTED** [review]

File: how-to-docs.html (hunk 8 above)
Comment (jwakely) [reply how-to-docs.html:81]: No idea! I think it might have been converted from something else.

File: how-to-docs.html (hunk 3 above)
Comment (jwakely) [reply how-to-docs.html:18]:
```suggestion
The <code>src</code> directory contains the code for reading the XML files and generating the HTML issues lists.
```

?

**Comment by jwakely** [comment]
Thanks, Casey! I'll squash all these suggestions before merging.

File: how-to-docs.html (hunk 3 above)
Comment (CaseyCarter) [reply how-to-docs.html:18]:
```suggestion
The <code>src</code> contains code that reads the XML files and generates the HTML issues lists.
```

**Comment by CaseyCarter** [comment]
Honestly we ought to rewrite this in markdown and merge it into the README, but it's probably not worth the effort given the size of our userbase.

**Review by CaseyCarter — APPROVED** [review]

**Comment by jwakely** [comment]
The only reason I bother updating it is because one day the next lwg chair is going to need to know this stuff. I think everybody who's already using this repo knows how it works.

**Pull request merged**